### DISCUSSION TITLE
The profile image; odd extra coding

### DISCUSSION THREAD

**Discussion by EAMorillon** [community/community#22702] [Other Feature Feedback, Questions, & Ideas]
<p>The profile image in my beginners website is not showing. I use the inspect element to look at the code and something odd is happening. <img src="https://user-images.githubusercontent.com/69782946/181072647-ad9981e5-a911-49a9-9289-222466287fe6.jpeg" alt="image code" data-base62-sha1="oS8XRPu86QfQz90nyePVkr7kFhi" width="386" height="97"></p>
<p>If you look at the code there is this extra code that i think is breaking the link “::before”. I think this is what’s causing the problem but i can’t fix it on my end. Could you please lend me a hand with this.</p>
<p>Best regards<br>
Edgar Morillon</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/eamorillon">@EAMorillon</a>, welcome to the GitHub Support Community!</p>
<p>It appears to be capitalization issue. Your images reside in the <strong>I</strong>mages folder (uppercase i), however, you’re referencing the <strong>i</strong>mages folder (lowercase i) which does not exist in your repo. The directory and file names are case-sensitive on GitHub’s servers, so the issue can be fixed in two ways:</p>
<ol>
<li>rename the folder from Images → images</li>
<li>or change the src from images/Edgar.jpg → Images/Edgar.jpg</li>
</ol>
<p>If you’re using Windows 10 <a class="mention" href="/tajmone">@tajmone</a> posted a solution for enabling case-sensitive behavior for files/folders, which you may want to check out. This way you can simulate how files/folders on GitHub’s servers work, and catch issues like these before pushing your changes to GitHub.</p><aside class="quote quote-modified" data-post="3" data-topic="127320">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/css-not-being-applied/127320/3">CSS Not being applied</a> <a class="badge-wrapper  bullet" href="/c/github-help/github-pages/31"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #c8e1ff;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Discussion and questions about GitHub Pages—websites for you and your projects hosted right on your GitHub repository.">GitHub Pages</span></a>
  </div>
  <blockquote>
    Hi <a class="mention" href="/mandrake-ux">@Mandrake-ux</a> and welcome on board! 
<a class="mention" href="/sethclydesdale">@sethclydesdale</a> is right, GH servers being Linux based have a case-sensitive file system. 
Since you didn’t spot the problem locally, I’m assuming you’re working under Windows OS. 
With Windows 10 you can now enable case-sensitive behavior on  a per-directory basis, which is always a good solution when working with cross-platform projects on GitHub: 

<a href="https://www.howtogeek.com/354220/how-to-enable-case-sensitive-folders-on-windows-10/" rel="noopener nofollow ugc">How-To-Geek » How to Enable Case Sensitive File and Folder Names on Windows 10</a>

I recommend you apply case-…
  </blockquote>
</aside>

<p>The ::before you saw is a <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/::before" rel="noopener nofollow ugc">CSS pseudo-element</a> and shouldn’t interfere with external resources. I’m assuming it’s simply added by your devtools, because Chrome’s (on my end) does not display it.</p>

**Comment by EAMorillon** [comment]
<p>Nice. I’ll make the change. I appreciate the help.</p>
